README.md

```diff
@@ -52,6 +52,59 @@ configure_opentelemetry(
 
 ```
 
+### Usage with Auto Instrumentation
+
+OpenTelemetry Python includes a command that allows the user to automatically instrument
+certain third party libraries. Here is an example that shows how to use this launcher
+with auto instrumentation.
+
+First, create a new virtual environment:
+
+```bash
+cd ~
+mkdir auto_instrumentation
+virtualenv auto_instrumentation
+source auto_instrumentation/bin/activate
+pip install opentelemetry-api
+pip install opentelemetry-sdk
+pip install opentelemetry-instrumentation
```

Comment:
these are not needed if `opentelemetry-launcher` is installed first (as it will install all of those as dependencies)

Reply:
Fixed